src/pkg/count/count.go

```diff
@@ -54,6 +66,18 @@ func (b *Bus) Add(k Key, n int64) {
 	}
 }
 
+// Add increases the count by n.
+// returns the current local value of k.
+func (b *Bus) AddRead(k Key, n int64) int64 {
```

Comment:
nit: `IncReadN` might a be more natural counterpart to `IncRead`?

Reply:
Eh, it's supposed to mimic the base func.  Ex:
```
Inc() -> IncRead()
Add() -> AddRead()
```

That said, I'm honestly not happy with the `FooRead()` name structure in the first place.  Suggestions?

Reply:
We could remove `FooRead` and have `Inc` and `Add` return the current value of counter. The caller can discard return val if not needed. Is this go kosher?

Reply:
it's certainly go kosher.  I actually initially went that route for implementation, then backed off because I wasn't sure I was happy with the two-op on every increment.  But, you know, we can probably eat that cost and be fine.